homeworks/ekaterina.sharova_rogerosha/PokemonProject/index.html

```diff
@@ -5,27 +5,49 @@
         <meta name="viewport"
             content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
         <meta http-equiv="X-UA-Compatible" content="ie=edge">
-        <title>PokemonProject</title>
+        <title>PokemonProject: About</title>
         <link
             href="./style.css"
             rel="stylesheet"
             type="text/css">
     </head>
     <body>
-        <h1 class="headline">About me</h1>
-        <p>
-            Hi! My name is Katya and I'm a Junior Frontend Developer. I am already familiar with main Web Technologies like
-            HTML, CSS, JavaScript, and Git version control system.
-        </p>
-        <p>
-            This page was developed during the course <a href="https://www.mastersacademy.education/frontend-for-beginners-it" class="link"> 'Frontend for beginners' from Masters Academy in 2023</a>.
-        </p>
-        <p>
-            This is a social project from MOCG company where I got an opportunity to work with Frontend mentors and to create my
-            own small project for the portfolio.
-        </p>
-        <p>
-            You can contact me via telegram @rogerosha and/or check out my GitHub.
-        </p>
+        <header class="upper-header">
+            <div class="ma-logo">
+                <img class="MAlogo" src="images/MA-logo.svg" alt="Masters Academy">
+            </div>
+            <ul class="navigation">
+                <li>
+                    <a href="./pokemons.html" class="navigation-item pokemons">Pokemons</a>
+                </li>
+                <li>
+                    <a href="#" class="navigation-item about">About</a>
+                </li>
+            </ul>
+        </header>
+        <div class="content">
+            <div class="yellow-rectangle">
```

Comment:
You can export it in one piece, as you've done for logo
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/476272dc-2505-46f1-95a1-0ec52dff1fe6)

Reply:
Finally I found the way to save it together! I thought it's a part of the homework to place them both at the same page 😄